src/utils/github/getPRsPerSHAS.ts

```diff
@@ -7,25 +8,99 @@ export default async function getPRsPerSHAS({
   owner,
   repo,
   shaArray,
+  repoSource,
 }: {
   email: string;
   owner: string;
   repo: string;
   shaArray: string;
+  repoSource: string;
 }) {
-  const issues = await axios
-    .post(`${backendURL}/api/github/getIssuesByCommits`, {
-      user: email,
-      repo,
-      owner,
-      commitList: shaArray
-    })
-    .then((res) => res.data)
-    .catch((err) => {
-      let reporter = analyticsReporter();
-      console.log("err getprsperSHAS", err);
-      let { message } = err;
-      reporter?.sendTelemetryException(err, { error: message });
-    });
-    return issues.items;
+  let issuesItems;
+
+  if (repoSource === "github.com") {
+    const issues = await axios
+      .post(`${backendURL}/api/github/getIssuesByCommits`, {
+        user: email,
+        repo,
+        owner,
+        commitList: shaArray,
+      })
+      .then((res) => res.data)
+      .catch((err) => {
+        let reporter = analyticsReporter();
+        let { message } = err;
+        reporter?.sendTelemetryException(err, { error: message });
+      });
+    issuesItems = issues.items;
+  } else if (repoSource === "gitlab.com") {
```

Comment:
probs a switch statement looks better.
We will have enterprise github and private Gitlabs

Reply:
Not in the short term but I can change the syntax